api/src/main/java/org/openmrs/validator/PersonAddressValidator.java

```diff
@@ -66,7 +66,7 @@ public void validate(Object object, Errors errors) {
 		if (object == null) {
 			System.out.println("The personAddress object should not be null");
 
-			System.exit(0); // Terminates JVM
+			return 0 ; // Terminates JVM
```

Comment:
It does not compile

Reply:
This sounds reasonable: 
errors.reject("error.name");
return;

Reply:
@achilep your trying to return a value `0` from a void method hence causing the compilation error.